sidecar/main.go

```diff
@@ -106,11 +119,22 @@ func main() {
 		PodNamespace: *podNamespace,
 		PodUID:       *podUID,
 	}
-	err = nodeMgr.Deploy()
+
+	nodeObj, err := nodeMgr.GetCSIAddonsNode()
```

Comment:
if `nodeObj` is used in the the different function but in the same package we dont need to expose it outside.

Reply:
Not sure I understand. `GetCSIAddonsNode` was made public as it sits in a different package.

Reply:
as i can see its not used by any other functions outside of `csiaddonsnode.go` except main (even its not required) so its not required to be done in such way. `DispatchWatcher` can call the internal `getCSIAddonsNode` and use it